<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WINDOWS-A1KASTV" ignored="0" name="Test" tests="5" failures="1" timestamp="2022-04-19T16:36:11 IST" time="0.101" errors="0">
  <testcase name="task1" time="0.015" classname="com.testNG.ann.Demo"/>
  <testcase name="task2" time="0.0" classname="com.testNG.ann.Demo">
    <skipped/>
  </testcase> <!-- task2 -->
  <testcase name="task2" time="0.0" classname="com.testNG.ann.Demo">
    <skipped/>
  </testcase> <!-- task2 -->
  <testcase name="task2" time="0.0" classname="com.testNG.ann.Demo">
    <skipped/>
  </testcase> <!-- task2 -->
  <testcase name="task2" time="0.0" classname="com.testNG.ann.Demo">
    <failure type="java.lang.AssertionError" message="expected [5678] but found [1234]">
      <![CDATA[java.lang.AssertionError: expected [5678] but found [1234]
at com.testNG.ann.Demo.task2(Demo.java:17)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- task2 -->
</testsuite> <!-- Test -->
